packages/eas-cli/src/utils/fingerprintCli.ts

```diff
@@ -13,14 +14,30 @@ export type FingerprintOptions = {
   cwd?: string;
 };
 
+export function diffFingerprint(
+  projectDir: string,
+  fingerprint1: Fingerprint,
+  fingerprint2: Fingerprint
+): FingerprintDiffItem[] | null {
+  // @expo/fingerprint is exported in the expo package for SDK 52+
+  const fingerprintPath = silentResolveFrom(projectDir, 'expo/fingerprint');
+  if (!fingerprintPath) {
+    return null;
+  }
+
+  const Fingerprint = require(fingerprintPath);
+  return Fingerprint.diffFingerprints(fingerprint1, fingerprint2);
```

Comment:
add a test in `@expo/fingerprint` package to ensure that that this function signature remains stable, similar to what we have for `Fingerprint.createFingerprintAsync`.

Reply:
https://github.com/expo/expo/pull/33410